libraries/src/MVC/Model/AdminModel.php

```diff
@@ -1254,6 +1254,11 @@ public function save($data)
                 return false;
             }
 
+            if ($context == 'com_plugins.plugin')
+            {
```

Comment:
```suggestion
            if ($context === 'com_plugins.plugin') {
```
Code style and type safe comparison.